libmscore/beam.cpp

```diff
@@ -2141,8 +2141,8 @@ void Beam::editDrag(EditData& ed)
             y1 += dy;
 
       qreal _spatium = spatium();
-      undoChangeProperty(Pid::BEAM_POS, QPointF(y1 / _spatium, y2 / _spatium));
       undoChangeProperty(Pid::USER_MODIFIED, true);
+      undoChangeProperty(Pid::BEAM_POS, QPointF(y1 / _spatium, y2 / _spatium));
```

Comment:
Are there any reasons why we need this order exactly? If so it would be better to leave a comment explaining that in the code.

Reply:
Yes, there is a reason. See https://musescore.org/en/node/289254#comment-933890, which is in response to the comment just before it. I have just added a comment in the code explaining why the order is important.

Reply:
OK, thanks!